### DISCUSSION TITLE
Error in SkyWalking installation using Helm

### DISCUSSION THREAD

**Discussion by roshnithadevus27** [apache/skywalking#12025] [Q&A]
Hi

We tried t install the skywalking in kubctl using the helm chart. 
We are followed the following link:https://github.com/apache/skywalking-helm .
But we are unable to make it up. 
We used the following command to install  <COPY THE INSTALLTION COMMAND USED>.It executed successfully. But when we tried to check the status and all we are getting the following error .Did we miss something. Attaching the screen shot below. Please help us on this. Is there any proper documentation available for the same. If so can you please share that as well. Thanks in advance
https://github.com/apache/skywalking-helm/tree/master

<img width="666" alt="MicrosoftTeams-image (21)" src="https://github.com/apache/skywalking/assets/156880868/d7b8e976-af68-4b39-8be9-3cf9bc302e67">

**Comment by wu-sheng** [comment]
We never face issue.
Please use released version.

https://github.com/apache/skywalking-helm/tree/v4.5.0

Helm scripts are verified through CI.

https://github.com/apache/skywalking-helm/blob/master/.github/workflows/e2e.ci.yaml


Your error should highly relate with your env, it is better you could understand the component relationship, and check the logs accordingly.

**Comment by wu-sheng** [comment]
8080 refused by target is not a thing we did.

**Comment by Superskyyy** [comment]
I think theres a networking issue in your environment. Try looking at proxies etc.
